client/blocks/upsell-nudge/index.jsx

```diff
@@ -56,11 +56,14 @@ export const UpsellNudge = ( {
 		return null;
 	}
 
-	if ( ! href && site ) {
-		href = addQueryArgs( { feature, plan }, `/plans/${ site.slug }` );
-	}
+	const customerType = plan === 'business-bundle' ? 'business' : 'personal';
```

Comment:
I'd suggest to use the `PLAN_BUSINESS` constant instead of having hardcoded the value.
```suggestion
	const customerType = plan === PLAN_BUSINESS ? 'business' : 'personal';
```

(Note that you'd need to import it first: `import { PLAN_BUSINESS } from 'lib/plans/constants';`)

Also, is there any reason why this defaults to `personal`? What if the given plan is neither Business or Personal ([example](https://github.com/Automattic/wp-calypso/blob/b081c32d30efdfdcb40119808ff5a9666477bfbf/client/my-sites/activity/activity-log-banner/upgrade-banner.jsx))? Or if no plan is given ([example](https://github.com/Automattic/wp-calypso/blob/b081c32d30efdfdcb40119808ff5a9666477bfbf/client/components/tinymce/plugins/simple-payments/dialog/index.jsx#L477-L488))?

Reply:
> I'd suggest to use the PLAN_BUSINESS constant instead of having hardcoded the value.

That reminds me to make this work with the eCommerce plan too! I'm not sure if there's any examples of an eCommerce nudge yet, but I've made it work with that too. 

(It can be tested by switching `PLAN_BUSINESS` to `PLAN_ECOMMERCE` with one of the nudges)

Anyway, I've updated that now! 

> Also, is there any reason why this defaults to personal? What if the given plan is neither Business or Personal (example)? Or if no plan is given (example)?

Just to be clear, `personal` (along with `business`) in this case doesn't refer to the plan name; it refers to the Personal and Premium plan which is displayed on the Plans page.

https://github.com/Automattic/wp-calypso/blob/6835546ab0c857b638afaaa7226d07390be8e092/client/my-sites/plans-features-main/index.jsx#L308

Currently, the customerType refers to the following:

- `personal`: Personal/Premium Plans
- `business`: Premium/Business/eCommerce Plans

This PR would remove Premium from the `business` customerType but _only_ if a nudge has immediately been clicked - there are no wider changes overall. This is ideal because otherwise people would click a nudge for a feature and see the Premium Plan, when in reality, that feature would require a Business or eCommerce plan.

As such, in the examples you've mentioned on the Activity Log and Simple Payments, nothing would change with this PR: people would still be offered the Personal and Premium plan after clicking the nudge. :)

Reply:
> As such, in the examples you've mentioned on the Activity Log and Simple Payments, nothing would change with this PR: people would still be offered the Personal and Premium plan after clicking the nudge. :)

The problem with my examples is that they are not actually regular but Jetpack plans. But don't worry, I'll make sure to test them 😉